modules/social_features/social_profile/social_profile.post_update.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Social Profile.
+ */
+
+use Drupal\profile\Entity\ProfileInterface;
+
+/**
+ * Update Profile names.
+ */
+function social_profile_post_update_0001(&$sandbox) {
+  /** @var \Drupal\profile\ProfileStorageInterface $profile_storage */
+  $profile_storage = \Drupal::entityTypeManager()->getStorage('profile');
+
+  if (!isset($sandbox['count'])) {
+    $sandbox['ids'] = \Drupal::entityQuery('profile')
+      ->condition('type', 'profile')
+      ->accessCheck(FALSE)
+      ->execute();
+    $sandbox['count'] = count($sandbox['ids']);
+  }
+
+  $ids = array_splice($sandbox['ids'], 0, 50);
+
+  foreach ($ids as $id) {
```

Comment:
Shall we be able to loadMultiple here?

Reply:
Yes, you right. Forgot to change in this place. Will be done.

Reply:
Thanks 👍🏾

Reply:
Done.